build-support/bin/generate_github_workflows.py

```diff
@@ -126,9 +127,13 @@ def pants_virtualenv_cache() -> Step:
     }
 
 
-def global_env() -> Env:
+def global_env(*, disable_buildsense: bool = False) -> Env:
     return {
-        "PANTS_CONFIG_FILES": "+['pants.ci.toml']",
+        "PANTS_CONFIG_FILES": (
+            "+['pants.ci.toml']"
+            if not disable_buildsense
```

Comment:
nit: I'd write this as `"+['pants.ci.toml'] if disable_buildsense else "+['pants.ci.toml', 'pants.no-buildsense.toml']"`